_posts/2023-01-14-release-2023.01.14.md

```diff
@@ -0,0 +1,136 @@
+
+We don't have a new release naming scheme yet, but something similar to GNOME again is discussed. For example, having a major number change when new features are introduced, and minor number changes for bug fixing.
+
+This will be the last release with this formatting, so everyone say goodbye to this numbering scheme which was there since the beginning of Bottles.
+
+Even with this change, Bottles development will not stop, far from it. It will continue to be developped by our developers and contributors, and don't hesitate to create a pull request or even issues if you have suggestions or if you run into problems.
+
+With this change, we also decided to focus our next couple releases to bug fixing and polishing Bottles. There won't be big visual changes and big new features for two or three releases, until we're happy with the state of Bottles. Bottles will feel all the better after this, and we will be able to add new features without breaking a lot of stuff in the future!
+
+## Changes for 2022.12.14
+### Library revamp
+The library got a new design! Deleting a card can also be undone.
+
+<div class="grid-pics">
+    <img tooltip="Library view" class="on-dark" src="/uploads/bottles-new-library-hover-dark.png" />
+    <img tooltip="Library view after delete" class="on-dark" src="/uploads/bottles-new-library-delete-dark.png" />
+</div>
+
+<div class="grid-pics">
+    <img tooltip="Library view" class="on-light" src="/uploads/bottles-new-libray-hover-light.png" />
+    <img tooltip="Library view after delete" class="on-light" src="/uploads/bottles-new-libray-delete-light.png" />
+</div>
+
+### New Window manager decorations toggle
+There's now a new settings to disable window decorations in the `Display Settings` of each bottle. This fixes issue [#2309](https://github.com/bottlesdevs/Bottles/issues/2309) where the settings was only available in Wine config.
+
+<img tooltip="Display Settings View" class="on-light" src="/uploads/bottles-new-toggle-win-decs-light.png" />
+<img tooltip="Display Settings View" class="on-dark" src="/uploads/bottles-new-toggle-win-decs.png" />
+
+### New runtime to support latest EAC
+We also updated the runtime used by Bottles with the latest version of Easy Anti-Cheat libraries! This should help with some games that requires this anti-cheat software.
+
+
+### New FSR settings
+The options window for FidelityFX Super Resolution was updated to be clearer about what is does.
+
+<img tooltip="Display Settings View" class="on-light" src="/uploads/bottles-new-fsr-settings.png" />
+<img tooltip="Display Settings View" class="on-dark" src="/uploads/bottles-new-fsr-settings-dark.png" />
+
+### GStreamer related changes
+Some runners ships with GStreamer plugins, which is great! But it wasn't properly used by Bottles. There were some changes to the backend so that now it is properly used, and works in any circumstances. You can have more details in the [FAQ](https://docs.usebottles.com/faq/video-gstreamer-problems).
```

Comment:
```suggestion
Some runners ship with GStreamer plugins, which is great! But it wasn't properly used by Bottles. There were some changes to the backend so that now it is properly used, and works in any circumstances. You can have more details in the [FAQ](https://docs.usebottles.com/faq/video-gstreamer-problems).
```